src/cli.rs

```diff
@@ -16,20 +16,21 @@ pub fn build_cli() -> App<'static, 'static> {
         )
         .subcommand(
             SubCommand::with_name("install")
+                .setting(AppSettings::TrailingVarArg)
+                .setting(AppSettings::AllowLeadingHyphen)
                 .about("Installs a specific Ruby version")
                 .arg(
                     Arg::with_name("list")
                         .short("l")
                         .long("list")
                         .help("Lists Ruby versions available to install"),
                 )
+                .arg(Arg::with_name("version").index(1))
                 .arg(
-                    Arg::with_name("with-openssl-dir")
-                        .short("w")
-                        .long("with-openssl-dir")
-                        .help("Specify a openssl directory"),
-                )
-                .arg(Arg::with_name("version").index(1)),
+                    Arg::with_name("configure_opts")
+                        .multiple(true)
+                        .help("Options passed to Ruby's ./configure"),
```

Comment:
The single quotation in `Ruby's` breaks completions files. I got the following error when I try using completions. In order to solve this problem, could you remove `Ruby's`?

```suggestion
                        .help("Options passed to ./configure"),
```

```
_frum:178: unmatched '
```